DeepNeuralNetworks/DNN.ipynb

```diff
@@ -1,1399 +1,2548 @@
 {
```

Comment:
<p>I don't think you need to write it out in a paragraph - if you just commit the lines of the model construction below I think that's enough </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>